src/main/resources/assets/spectritemod/lang/de_de.lang

```diff
@@ -0,0 +1,228 @@
+tile.iron_chest.name=Eisentruhe
+tile.gold_chest.name=Goldtruhe
+tile.diamond_chest.name=Diamanttruhe
+tile.spectrite_chest.name=Spectrittruhe
+tile.large_spectrite_chest.name=Große Spectrittruhe
+tile.spectrite_chest_trapped.name=Gesicherte Spectrittruhe
+tile.large_spectrite_chest_trapped.name=Große Gesicherte Spectrittruhe
+tile.spectrite_ore_surface.name=Spectriterz
+tile.spectrite_ore_nether.name=Netherrack-Spectriterz
+tile.spectrite_ore_end.name=Endstein-Spectriterz
+tile.spectrite_block.name=Spectritblock
+item.diamond_rod.name=Diamantstange
+item.spectrite_rod.name=Spectritstange
+item.spectrite_gem.name=Spectritstein
+item.spectrite_orb.name=Heilende Spectritkugel
+item.spectrite_shovel.name=Spectritschaufel
+item.spectrite_shovel_special.name=Perfekte Spectritschaufel
+item.spectrite_pickaxe.name=Spectritspitzhacke
+item.spectrite_pickaxe_special.name=Perfekte Spectritspitzhacke
+item.spectrite_axe.name=Spectritaxt
+item.spectrite_axe_special.name=Perfekte Spectritaxt
+item.spectrite_sword.name=Spectritschwert
+item.spectrite_sword_2.name=Legendäre Spectritklinge
+item.spectrite_sword_special.name=Perfektes Spectritschwert
+item.spectrite_sword_2_special.name=Perfekte Legendäre Spectritklinge
+item.spectrite_helmet.name=Spectrithelm
+item.spectrite_chestplate.name=Spectritbrustpanzer
+item.spectrite_leggings.name=Spectritbeinschutz
+item.spectrite_boots.name=Spectritstiefel
+iteminfo.spectrite_orb.l1=§7Alle #s:
+iteminfo.spectrite_orb.l2=§7Regeneration I (#s)
+iteminfo.spectrite_orb.l3=§7(Erhöht sich mit Spectritrüstung)@
+iteminfo.spectrite_shovel.l1=§7Alle #s:
+iteminfo.spectrite_shovel.l2=§7Baut 5 Blöcke ab@
+iteminfo.spectrite_shovel_special.l1=§7Alle #s:
+iteminfo.spectrite_shovel_special.l2=§7Baut 9 Blöcke ab@
+iteminfo.spectrite_pickaxe.l1=§7Alle #s:
+iteminfo.spectrite_pickaxe.l2=§7Baut 5 Blöcke ab@
+iteminfo.spectrite_pickaxe_special.l1=§7Alle #s:
+iteminfo.spectrite_pickaxe_special.l2=§7Baut 9 Blöcke ab@
+iteminfo.spectrite_axe.l1=§7Alle #s:
+iteminfo.spectrite_axe.l2=§7Baut 5 Blöcke ab@
+iteminfo.spectrite_axe_special.l1=§7Alle #s:
+iteminfo.spectrite_axe_special.l2=§7Baut 9 Blöcke ab@
+iteminfo.spectrite_sword.l1=§7Alle #s:
+iteminfo.spectrite_sword.l2=§7Sofortschaden I
+iteminfo.spectrite_sword.l3=§7Giftschaden I@
+iteminfo.spectrite_sword_special.l1=§7Alle #s:
+iteminfo.spectrite_sword_special.l2=§7Sofortschaden II
+iteminfo.spectrite_sword_special.l3=§7Giftschaden II@
```

Comment:
Original: "Disperse Damage"
Direct translation: "Verstreuter/Verteilter Schaden" which sounds bad.
Changed it to "Giftschaden" ("Poison damage") for now. 
Ideas?